HLTrigger/Configuration/python/HLT_75e33/modules/hltFixedGridRhoFastjetAllCaloForMuons_cfi.py

```diff
@@ -1,7 +1,13 @@
 import FWCore.ParameterSet.Config as cms
 
-hltFixedGridRhoFastjetAllCaloForMuons = cms.EDProducer("FixedGridRhoProducerFastjet",
+hltFixedGridRhoFastjetAllCaloForMuons = cms.EDProducer("FixedGridRhoProducerFastjetFromRecHit",
     gridSpacing = cms.double(0.55),
     maxRapidity = cms.double(2.5),
-    pfCandidatesTag = cms.InputTag("hltTowerMakerForAll")
+    hbheRecHitsTag = cms.InputTag( "hltHbhereco" ),
+    ebRecHitsTag = cms.InputTag( 'hltEcalRecHit','EcalRecHitsEB' ),
+    eeRecHitsTag = cms.InputTag( 'hltEcalRecHit','EcalRecHitsEE' ),
+    skipHCAL = cms.bool( False ),
+    skipECAL = cms.bool( False ),
+    eThresHB = cms.vdouble( 0.1, 0.2, 0.3, 0.3 ),
+    eThresHE = cms.vdouble( 0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 )
```

Comment:
These look like the Run-3 thresholds. Are they the same for Phase 2 ?

Reply:
Let's double check this with @cms-sw/hcal-dpg-l2 and @cms-sw/pf-l2 
Is the following still valid for phase2? Or do we have new PF recHit threshold for HB for phase2?
https://github.com/cms-sw/cmssw/blob/33ef05739ed403d4af6ba915a406905080dbc5ca/RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py#L5

Reply:
I thought this depends on the 'aging' scenario, e.g.
https://github.com/cms-sw/cmssw/blob/af4b7abc8eb5bf1bcf2dd39e576e76ea1bd0b464/SLHCUpgradeSimulations/Configuration/python/aging.py#L96
, and it would match what is used in the modules below [*], where there are already some differences.

Indeed, it would be good to clarify this.

[*]

https://github.com/cms-sw/cmssw/blob/af4b7abc8eb5bf1bcf2dd39e576e76ea1bd0b464/HLTrigger/Configuration/python/HLT_75e33/modules/particleFlowRecHitHBHE_cfi.py#L16
https://github.com/cms-sw/cmssw/blob/af4b7abc8eb5bf1bcf2dd39e576e76ea1bd0b464/HLTrigger/Configuration/python/HLT_75e33/modules/particleFlowClusterHBHE_cfi.py#L13

https://github.com/cms-sw/cmssw/blob/af4b7abc8eb5bf1bcf2dd39e576e76ea1bd0b464/HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitHBHEForEgamma_cfi.py#L25
https://github.com/cms-sw/cmssw/blob/af4b7abc8eb5bf1bcf2dd39e576e76ea1bd0b464/HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHBHEForEgamma_cfi.py#L13

Reply:
I was about to point to usual Phase2 aging customization
https://github.com/cms-sw/cmssw/blob/master/SLHCUpgradeSimulations/Configuration/python/aging.py#245 
With PFlowClusters thresholds/seeds involved.
But just noticed that Marino has already commented on it _several minutes ago_...